src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -128,6 +130,41 @@ trait BackendCall {
     throw new NotImplementedError(s"resume() called on a non-resumable BackendCall: $this")
   }
 
+  def useCachedCall(cachedBackendCall: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] = {
+    throw new NotImplementedError(s"useCachedCall() is not implemented for BackendCall: $this")
```

Comment:
Same comment that I made on @Horneth 's PR. If all you're going to say is "foo is not implemented" just make it a ??? as that seems pretty redundant. It's petty semantics but saying "not supported" doesn't actually rub me the wrong way though